doc/security/use_cases/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "OpenTitan Use Cases"
+---
+
+
+## Overview
+
+OpenTitan’s mission is to raise the security bar industry wide by implementing a transparent, logically secure hardware root of trust with wide application.
+
+This document describes some of those use cases for OpenTitan.
+These range from data center integrations to implement platform integrity solutions, to embedded security applications such as security keys and smart cards.
```

Comment:
Is "data center integrations to implement platform integrity solutions" one thing? This sentence could probably be split up into two to make it less dense and easier to understand.

Reply:
agree with @imphil .  I feel like data center integrity solutions is probably sufficient.

Reply:
Done.